src/Backend/Modules/Location/Actions/Edit.php

```diff
@@ -47,8 +47,11 @@ public function execute(): void
                 $this->redirect(BackendModel::createUrlForAction('Index', 'Settings'));
             }
 
-            // add js
-            $this->header->addJS('https://maps.googleapis.com/maps/api/js?key=' . $apikey);
+            $interfaceLanguage = BL::getInterfaceLanguage();
+            $googleLanguage = $interfaceLanguage . '_' . strtoupper($interfaceLanguage);
```

Comment:
Where did you find the information that states the language should be in the `nl_NL` format?

I found it's not necessary ( https://developers.google.com/maps/documentation/javascript/localization ) which is fortunate because this code will generate stuff like `en_EN` which does not exist.

Reply:
Changed in bb88e9f